src/test/java/io/stargate/sgv2/jsonapi/api/v1/AbstractNamespaceIntegrationTestBase.java

```diff
@@ -99,6 +103,47 @@ protected int getTestPort() {
     }
   }
 
+  protected Map<String, ?> getHeaders() {
+    if (useDseCql()) {
+      return Map.of(
+          HttpConstants.AUTHENTICATION_USER_NAME_HEADER,
+          getCassandraUsername(),
+          HttpConstants.AUTHENTICATION_PASSWORD_HEADER,
+          getCassandraPassword(),
+          HttpConstants.EMBEDDING_AUTHENTICATION_TOKEN_HEADER_NAME,
+          CustomITEmbeddingProvider.TEST_API_KEY);
+    } else {
+      return Map.of(
+          HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME,
+          getAuthToken(),
+          HttpConstants.EMBEDDING_AUTHENTICATION_TOKEN_HEADER_NAME,
+          CustomITEmbeddingProvider.TEST_API_KEY);
+    }
+  }
+
+  protected Map<String, ?> getInvalidHeaders() {
+    if (useDseCql()) {
+      return Map.of(
+          HttpConstants.AUTHENTICATION_USER_NAME_HEADER,
+          "invalid",
+          HttpConstants.AUTHENTICATION_PASSWORD_HEADER,
+          getCassandraPassword(),
+          HttpConstants.EMBEDDING_AUTHENTICATION_TOKEN_HEADER_NAME,
+          CustomITEmbeddingProvider.TEST_API_KEY);
+    } else {
+      return Map.of(
+          HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME,
+          "invalid",
+          HttpConstants.EMBEDDING_AUTHENTICATION_TOKEN_HEADER_NAME,
+          CustomITEmbeddingProvider.TEST_API_KEY);
+    }
+  }
+
+  protected boolean useDseCql() {
```

Comment:
I find this operation name confusing. I think the purpose of this is to determine whether we are running a coordinator or not? If we are not using a coordinator, then I agree we'd use the "DSE" node as the place to connect for CQL, and I guess that is where the name comes from ("use DSE for CQL"?). I feel it would be easier to understand if we have an explicit property that says whether to run a coordinator or not.